### ISSUE TITLE
gogs-webhook didn't support gogs release event

### ISSUE THREAD

**Issue by sagaii** [jenkinsci/gogs-webhook-plugin#45]
- Gogs version (or commit ref):
- Jenkins version: Jenkins ver. 2.107.3
- Jenkins modules used in combination with gogs-webhook-plugin: 
- JSON Output of webhook: {"result":"ERROR","message":"Only push event can be accepted."}

## Description
I had  tried to config my gogs system to support auto release my test version to work version.  
so i hope in gogs system create one repo version , it will send out one webhook event . in fact, gogs had 
support the funciton. i had choosed event of repo release . it give out this post data:

Request URL: http://jenkins.address/gogs-webhook/?job=releasedb/copytest
Request method: POST
Content-Type: application/json
X-Gogs-Delivery: 7f96dbfc-c93a-4247-8528-0c7e1566c53d
X-Gogs-Event: release
X-Gogs-Signature: 03cfb231c9c643277fae62bd318538e9e77157c630cb1373e7f37ab84cc14e94
  

I had read  gogs=webhook source code , in webhook.java  , it only receive "push" event. 
so , i hope it can add "release" and other's event. 

thx. 

...

**Comment by sanderv32** [comment]
@sagaii what should the release event do other then triggering a job?

**Comment by sagaii** [comment]
Gogs-webhook receive release event ,   triggering a job.  and the job can execute deploy work,t example copy program from test environment to production environment.

**Comment by sagaii** [comment]
in this gogs-webhook didn't support release event. so i only use build token root plugin  , create a url link to  triggering a job.

**Comment by sagaii** [comment]
yes, i think so .   it same as a push event.   but i need it . :) .

**Comment by schlitzered** [comment]
hello i would also need a feature like this, the jenkins plugin should react on tag creation, reacting on release creation would also work, since it basically creates a tag.

**Comment by fseidler** [comment]
Hello
I also need this feature, I have different jobs that should run based on the type of the event: release or push

**Comment by aeciopires** [comment]
+1

**Comment by trrtly** [comment]
+1

**Comment by toan267** [comment]
+1

**Comment by sanderv32** [comment]
Opened PR #62 for this.
